plugins/commando.c

```diff
@@ -1041,6 +1041,27 @@ static struct command_result *listdatastore_done(struct command *cmd,
 	return command_finished(cmd, js);
 }
 
+static bool blacklist_merge(struct blacklist *entry, struct blacklist *blacklist, struct blacklist *nextblacklist) 
+{
+	bool nextmerged = false;
```

Comment:
I prefer to avoid this "initialize, then override" pattern when I can.  Because if you only initialize this below:

```
}
	if (blacklist->end >= (nextblacklist->start -1)) {
		blacklist->end = nextblacklist->end;
		nextmerged = true;
	} else {
		nextmerged = false;
}
```

Then if the code ever changes, the compiler will warn that nextmerge is not set when you return it.